File: src/javascript_implementation/search/linear_search.js
const array = [1, 2, 3, 4, 5];
var target = 5;

function linearSearch(array, target) {
    for (var i = 0; i < array.length; i++) {
        if (array[i] === target) {
            return i;
        }
    }
    return -1;
}

console.log(linearSearch(array, target));

File: src/java_implementation/search/LinearSearch.java
package src.java_implementation.search;

public class LinearSearch {
    public static void main(String[] args) {
        int[] array = {1, 2, 3, 4, 5};
        int target = 5;

        System.out.println(linearSearch(array, target) != -1 ? "Found" : "Not found");
    }   

    public static int linearSearch(int array[], int target) {
        for (int i = 0; i < array.length; i++) {
            if (array[i] == target) {
                return i;
            }
        }

        return -1;
    }
}
